internal/util/util.go

```diff
@@ -141,3 +144,13 @@ func GBToTB[T ~int | ~int64 | ~float64](gb T) float64 {
 func MBToGB[T ~int | ~int32 | ~float64](mb T) int {
 	return int(math.Round(float64(mb) / 1024.0))
 }
+
+func GetInventoryVersion(inventory []byte) int {
```

Comment:
For future reference
Please add here a comment - what is the difference between v1 to v2
Why do we check it this way
Maybe also add a link to the PR where we changed from V1 to V2